BUILDING_ON_LINUX.md

```diff
@@ -65,6 +65,23 @@ _Most likely works the same for other Red Hat-like distros. Substitute `dnf` wit
 
 `qmake-qt5 .. && make -j$(nproc)`
 
+## openSUSE Tumbleweed
+
+_Not tested in Leap, but the packages should be the same._
+
+1. Install all of the dependencies using `sudo zypper install libopenssl-devel libqt5-qtbase-devel libqt5-qtmultimedia-devel libqt5-qtsvg-devel libboost_filesystem-devel libboost_system-devel libsecret-devel boost-devel cmake`
+1. Go into the project directory
+1. Create a build folder and go into it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
+
+### Using CMake
+
+`cmake .. && make -j$(nproc)`
```

Comment:
I can't figure out why it's using the wrong linker (`/usr/lib64/gcc/x86_64-suse-linux/8/../../../../x86_64-suse-linux/bin/ld`) instead of the `g++-8` compiler itself. Someone with more CMake knowledge would have to look at that one.